extensions/amp-base-carousel/1.0/base-carousel.js

```diff
@@ -319,6 +320,19 @@ function BaseCarouselWithRef(
       }}
       tabIndex="0"
       wrapperClassName={classes.carousel}
+      contentAs={lightbox ? WithLightbox : 'div'}
+      contentRef={contentRef}
+      contentProps={{
+        autoLightbox: false,
+        render: () =>
+          children.map((child) =>
+            child.props?.thumbnailSrc ? (
```

Comment:
Does this produces only thumbnails? Or the actual images as well for full display? If both: wouldn't `thumbnailSrc` be likely of a poor quality?

Reply:
That's a good point, removed.